### PR TITLE
LPS-189054 Create FDS Empty State example

### PR DESCRIPTION
**Pull request by edalgrin**
Follow up of https://github.com/liferay-frontend/liferay-portal/pull/3561

---

This PR adds FrontendDataSet component with no items in order to trigger the customized EmptyState view

image

### CODE REVIEW

**Comment by edalgrin** [comment]
ci:stop

File: modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/partials/empty.jsp [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	apiURL="<%= emptyFDSDisplayContext.getAPIURL() %>"
+	emptyState='<%=
+		HashMapBuilder.<String, Object>put(
+			"description", LanguageUtil.get(request, "start-creating-one-to-show-your-data")
+		).put(
+			"image", "/states/empty_state.gif"
+		).put(
+			"title", LanguageUtil.get(request, "no-data-sets-created")
+		).build()
+	%>'
```
Comment (dsanz) [inline modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/partials/empty.jsp:24]: Minor nit: empty state value can be returned by a Java method in `EmptyFDSDisplayContext` class

File: modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/java/com/liferay/frontend/data/set/taglib/servlet/taglib/HeadlessDisplayTag.java [hunk 2]
```diff
@@ -81,6 +81,10 @@ public CreationMenu getCreationMenu() {
 		return _creationMenu;
 	}
 
+	public Map<String, Object> getEmptyState() {
+		return _emptyState;
+	}
+
```
Comment (dsanz) [inline modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/java/com/liferay/frontend/data/set/taglib/servlet/taglib/HeadlessDisplayTag.java:87]: If we move all the added methods and attributes from `HeadlessDisplayTag` to `BaseDisplayTag` we'll get this feature for free in both tags. See for example how [we did with pagination](https://github.com/liferay/liferay-portal/commit/353b58ec49bbdfbf8cb548d20728ac152b546e7b). This is much simpler as there's no processing of the attribute.

We already did this from the beginning with the `uniformActionsDisplay` attribute, [here](https://github.com/liferay/liferay-portal/commit/f79018e8da5a4c2c8f2bd6008b4429d0aec45f10)

**Review by dsanz — CHANGES_REQUESTED** [review]

**Comment by dsanz** [comment]
LGTM, please see comments as we can make this available in the two tags with little effort, just applying a slightly different pattern using the same code

**Review by dsanz — APPROVED** [review]
LGTM ;)

**Comment by edalgrin** [comment]
ci:test:sf

**Comment by edalgrin** [comment]
ci:test:relevant

**Comment by mateusralv** [comment]
I'm taking a look 🕵🏽

**Comment by edalgrin** [comment]
ci:forward:force

**Pull request closed without merging**